src/__main__.py

```diff
@@ -88,20 +104,9 @@ async def list(ctx, code=None):
         code = code.upper().replace('-', '').replace(' ','')
         assignments = sheets_parser.fetch_assignments(service, SPREADSHEET_ID, RANGE_NAME)
         final_assignments = []
-        current_course = ''
-        current_code = ''
         # Remove all courses that don't have a matching course code and aren't within 14 days.
         for assignment in assignments:
-            if assignment.code != '':
```

Comment:
Good call removing the massive number of if statements here. The logic is already handled by `events.py`.